ntbs-service/Pages/Notifications/Edit/Episode.cshtml

```diff
@@ -51,43 +51,76 @@
     <br />
 
     <h2> Hospital details </h2>
-    <cascading-dropdowns model="Episode" shouldvalidatefull="@fullValidation" inline-template>
-        <div>
-            @{
-                var TBServiceCodeError = !Model.IsValid("Episode.TBServiceCode");
-                var TBServiceCodeGroupState = TBServiceCodeError ? Error : Standard;
-                var TBServiceCodeSelectErrorState = TBServiceCodeError ? SelectType.Error : SelectType.Standard;
-            }
-            <nhs-form-group ref="formGroupTBServiceCode" nhs-form-group-type=@TBServiceCodeGroupState>
-                <label nhs-label-type="Standard" asp-for="Episode.TBServiceCode">
-                    Notifying TB service (required)
-                </label>
-                <span nhs-span-type="ErrorMessage" id="tb-service-error" has-error="@TBServiceCodeError"
-                      ref="errorFieldTBServiceCode" asp-validation-for="Episode.TBServiceCode"></span>
-                <select ref="selectFieldTBServiceCode" nhs-select-type="@TBServiceCodeSelectErrorState" asp-for="Episode.TBServiceCode"
-                        asp-items="Model.TBServices" v-on:change="function (ev) {fetchHospitalList(ev); validate('TBServiceCode')(ev);}">
-                </select>
-            </nhs-form-group>
-            <br />
-            @{
-                var HospitalError = !Model.IsValid("Episode.HospitalId");
-                var HospitalGroupState = HospitalError ? Error : Standard;
-                var HospitalSelectErrorState = HospitalError ? SelectType.Error : SelectType.Standard;
-            }
-            <nhs-form-group ref="formGroupHospitalId" nhs-form-group-type=@HospitalGroupState>
-                <label nhs-label-type="Standard" asp-for="Episode.HospitalId">
-                    Hospital (required)
-                </label>
-                <span nhs-span-type="ErrorMessage" id="hospital-error" has-error="@HospitalError"
-                      ref="errorFieldHospitalId" asp-validation-for="Episode.HospitalId"></span>
-                <select ref="selectFieldHospitalId" nhs-select-type="@HospitalSelectErrorState" asp-for="Episode.HospitalId"
-                        asp-items="Model.Hospitals" v-on:change="function(ev) { validate('HospitalId')(ev); }">
-                    <option value="" selected>Select Hospital</option>
-                </select>
-            </nhs-form-group>
-            <br />
-        </div>
-    </cascading-dropdowns>
+
+    @if(Model.Notification.NotificationStatus == NotificationStatus.Draft) 

```

Comment:
quite a simple but large if else statement to show the 2 different behaviours

Reply:
We could probably modify `cascading-dropdowns` to behave sensibly when the first dropdown isn't present - that way the `if` could be just around the tbService control.

Having a glance at `cascading-dropdowns` it's very specific to this area, so we could specify that in the name too.

Reply:
Seems like a good idea especially if it helps with IDE being out of whack :)

Reply:
the dropdown behaves nicely already it turns out if you don't supply the first dropdown - we can just not call the function attached to the first dropdown, happy to rename to something a bit more specific